### ISSUE TITLE
[subviews] mortuary slab blending issue

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#607]
When you start the game and TNO is getting up from the slab, he is half-transparent. Afterwards this can be seen with him or Morte if you move them close enough. Or he's completely clipped:
![image](https://user-images.githubusercontent.com/121515/72216839-aa468880-3526-11ea-8279-2567b94cd05f.png)
![image](https://user-images.githubusercontent.com/121515/72216844-b7637780-3526-11ea-8862-a8ea30d53cf6.png)

Related to #441

**Comment by bradallred** [comment]
I'm guessing there are some flags or somesuch on those wall polygons that aren't being handled  in `Map::IntersectsWall`

**Comment by bradallred** [comment]
is this a `ScriptedAnimation`?

**Comment by MarcelHB** [comment]
"shade wall"-flag set according to NI.

![mortuary_clipping](https://user-images.githubusercontent.com/238558/72218686-0bc62180-353e-11ea-8d1b-3c18a86460b8.jpg)

**Comment by bradallred** [comment]
is a "Shade wall" just a normal wall? IESDP doesn't say anything about what that means.

**Comment by MarcelHB** [comment]
From how I read the display value, its the least significant bit set.

**Comment by bradallred** [comment]
I think the issue is a missing call to `Wall_Polygon::PointCovered`

**Comment by MarcelHB** [comment]
But let's look at the arch nearby:
![mortuary_clipping2](https://user-images.githubusercontent.com/238558/72219797-2e5e3780-354a-11ea-8f47-ca53559c46a4.jpg)

They have the same flag set. For the arch, any actor should go half-opaque, for the slab probably not. So what's missing?

**Comment by lynxlynxlynx** [comment]
The two wall groups don't have any flag difference, but the slab is not searchmap accessible.

**Comment by bradallred** [comment]
@MarcelHB I looked at the code and the conclusion I came to was that `WF_BASELINE` polygons have a "baseline" and if the actor pos (or pos + height?) is below the baseline than they are not occluded. For the arch I imagine then the baseline is simply low enough (not visible) that an actor is always above it.

We are missing this check currently and there are some other complexities to contend with to implement it. The old code never bothered with clipping for this check because that was incidental by virtue of drawing to a sprite rather than the screen.

I'll come back and look at this in a few weeks.
